FRPs/FRP-12.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 12
+title: MEV in l2 & sidechains exploration
+team: @obadiaa @fxfactorial
+created: 2021-06-04
+status: pending
+---
+
+# MEV in l2 & sidechains exploration
+
+
+## Background and Problem Statement
+As Ethereum's layer 1 becomes increasingly used, many applications are looking to move to outer layers of Ethereum in order to operate more efficiently (high tx throughput, lower tx fees). Each of these layers have different tx ordering rules from layer 1, different consensus mechanisms and different entities executing the ordering. Intuitively, it seems a lot of existing MEV transaction flow could move to these layers and makes MEV on such layers worth studying in further detail.
+
+In addition, a key property of Ethereum today widely used by most applications in DeFi is atomic composability: the ability to compose smart contracts together and execute multi-contract interactions atomically in a single transaction. However, atomic cross-layer composability does not seem to be achievable. In the case of DeFi, this implies potentially significant liquidity fragmentation which should be a large source of MEV. However, the mechanics of how it will be captured, and how layers will interact with each other is not clear at the moment.
+
+This initial FRP is an exploration of these themes to start wrapping our heads around the multi-chain multi-layered world.
+
+
+## Plan and Deliverables
+Since this is a rather large topic, we will prioritize l2s and sidechains with existing adoption and will limit work on the ones that are still actively changing their technology significantly.
```

Comment:
l2s -> L2s?

Reply:
"with existing adoption" -> "that show adoption"? or something else, "existing" sounds strange

Reply:
why limit to those changing their tech? if they have adoption, I'd still tackle them, perhaps even more so if they've solidified in terms of tech.

Reply:
I think I expressed myself badly, what this meant to say is that we'd focus the research on what already has adoption and where the tech isn't shifting significantly - to be as applied as possible